app/pages/page-list.ts

```diff
@@ -24,5 +25,10 @@ export var PAGE_LIST: Array<Page> = [
         title: "Amigo Requests",
         component: AmigoRequestsPage,
         icon: "person-add"
+    },
+    {
+        title: "My Medications",
+        component: MedListPage,
+        icon: "medkit"
```

Comment:
If this is going to use the `medkit` icon, then maybe we should change `Dose` above to `timer`.